core/src/main/java/com/sequenceiq/cloudbreak/converter/AmbariStackDetailsJsonToStackRepoDetailsConverter.java

```diff
@@ -59,19 +68,15 @@ public StackRepoDetails convert(AmbariStackDetailsJson source) {
         return repo;
     }
 
-    private void validateRepoFieldsExistence(AmbariStackDetailsJson source) {
-        boolean vdfRequiredFieldsExists = !StringUtils.isEmpty(source.getRepositoryVersion())
-                && !StringUtils.isEmpty(source.getVersionDefinitionFileUrl());
-
-        boolean customBaseRepoRequiredFieldsExists = !StringUtils.isEmpty(source.getStackBaseURL())
+    private boolean isCustomBaseRepoRequiredFieldsExists(AmbariStackDetailsJson source) {
+        return !StringUtils.isEmpty(source.getStackBaseURL())
```

Comment:
I suggest to use: `Stream.of(source.getStackRepoId(), ...).noneMatch(i -> StringUtils.isEmpty(i))`